packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -793,6 +793,7 @@ export default {
             const message = this.$t(error.messageKey) || this.$t('errorMessages.genericServerError');
             const eventToEmit = error.eventToEmit || EventNames.CheckoutFailure;
             const logMessage = error.logMessage || 'Consumer Checkout Failure';
+            const errorName = error.errorCode ? `${error.errorCode}-` : ''; // This appends the hyphen so it doesn't appear in the logs when the error name does not exist
```

Comment:
I wasn't a big fan of adding the hyphen here but the errorName is only used in one place and I'd probably have to write something a bit over the top to do it another way.